### ISSUE TITLE
Cordova Android 8.0.0 Warning

### ISSUE THREAD

**Issue by tehmaestro** [OneSignal/OneSignal-Cordova-SDK#543]
**Description:**

Hello, I updated my project from cordova-android 7.1.4 to 8.0.0, but I get a warning in the console:
```
WARNING: The onesignal-gradle-plugin MUST be before com.android.application!
   Please put onesignal-gradle-plugin first OR update to com.android.tools.build:gradle:3.0.0 or newer!
```

**Environment**

I am running:
```
Cordova CLI -> 8.1.2
Cordova Android -> 8.0.0
Node: 6.11.0
```
I created a new project just to test this. Same result.
I was using the following before the upgrade:
```
'com.android.tools.build:gradle:3.0.1'
gradleVersion = '4.1.0'
```

After updating to 8.0.0 I have:
```
'com.android.tools.build:gradle:3.3.0'
gradleVersion = '4.10.3'
```

**Steps to Reproduce Issue:**

1. cordova create my-app
2. cd my-app
3. cordova platform add android@8.0.0
4. cordova plugin add onesignal-cordova-plugin
5. cordova clean

This will output the warning.

Related to this: https://github.com/OneSignal/OneSignal-Cordova-SDK/issues/440
But I am using a gradle build tools higher than 3.0.0, I was already using a higher version before updating cordova-android.
As I mentioned, a newly created project will throw the warning. It seems like gradle plugin version is incorrectly detected maybe? Thank you so much for your help.

**Comment by sirlojik** [comment]
Same here. Even after applying build.gradle adjustments

```
apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'


buildscript {
    repositories {
        mavenCentral()
        google()
        jcenter()
		maven { url 'https://plugins.gradle.org/m2/' } // Gradle Plugin Portal 
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:3.3.0'
		classpath 'gradle.plugin.com.onesignal:onesignal-gradle-plugin:[0.12.5, 0.99.99]'
    }
}

```

**Comment by rgomezp** [comment]
TLDR; don't worry about this issue

Howdy,
Thanks for reporting this. We were able to reproduce, though this isn't really causing any issues that we're aware of. It's simply a warning at this point which you can just ignore. Unless this causes, say, a build failure, we will go ahead and close this issue. Feel free to open a new one if this starts causing serious issues.

**Comment by tehmaestro** [comment]
This actually makes the `clean` command fail (I think), or it does not complete successfully. `build` works, yes.

**Comment by Yerman3000** [comment]
I have the same problem. And it causes the App to generate an error with Android 9 only. How is it solved?

**Comment by Yerman3000** [comment]
> **Description:**
> 
> Hello, I updated my project from cordova-android 7.1.4 to 8.0.0, but I get a warning in the console:
> 
> ```
> WARNING: The onesignal-gradle-plugin MUST be before com.android.application!
>    Please put onesignal-gradle-plugin first OR update to com.android.tools.build:gradle:3.0.0 or newer!
> ```
> 
> **Environment**
> 
> I am running:
> 
> ```
> Cordova CLI -> 8.1.2
> Cordova Android -> 8.0.0
> Node: 6.11.0
> ```
> 
> I created a new project just to test this. Same result.
> I was using the following before the upgrade:
> 
> ```
> 'com.android.tools.build:gradle:3.0.1'
> gradleVersion = '4.1.0'
> ```
> 
> After updating to 8.0.0 I have:
> 
> ```
> 'com.android.tools.build:gradle:3.3.0'
> gradleVersion = '4.10.3'
> ```
> 
> **Steps to Reproduce Issue:**
> 
> 1. cordova create my-app
> 2. cd my-app
> 3. cordova platform add android@8.0.0
> 4. cordova plugin add onesignal-cordova-plugin
> 5. cordova clean
> 
> This will output the warning.
> 
> Related to this: #440
> But I am using a gradle build tools higher than 3.0.0, I was already using a higher version before updating cordova-android.
> As I mentioned, a newly created project will throw the warning. It seems like gradle plugin version is incorrectly detected maybe? Thank you so much for your help.

any solution?

**Comment by rgomezp** [comment]
@Yerman3000 ,
Please include a specific error and/or stack trace

**Comment by Yerman3000** [comment]
> @Yerman3000 ,
> Please include a specific error and/or stack trace

Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :app
WARNING: The onesignal-gradle-plugin MUST be before com.android.application!
   Please put onesignal-gradle-plugin first OR update to com.android.tools.build:gradle:3.0.0 or newer!

> Task :app:preBuild UP-TO-DATE
> Task :CordovaLib:preBuild UP-TO-DATE
> Task :CordovaLib:preDebugBuild UP-TO-DATE
> Task :CordovaLib:checkDebugManifest UP-TO-DATE
> Task :CordovaLib:processDebugManifest UP-TO-D

----------------------------------------------

the apk works fine. But in Android 9  no works. 
My Cordova version is 9.0.0 (cordova-lib@9.0.1)

**Comment by rgomezp** [comment]
Howdy @Yerman3000 ,
How are you building the project? Are you using PGB?

**Comment by Yerman3000** [comment]
Por tu nombre podrías hablar en español?.
Desde que actualizaron el plugin el apk solo   no funciona con android 9.. Si le quito el plugin anda bien en todas las versiones

**Comment by rgomezp** [comment]
Hola que tal,
Checa #385
